parts/linux/cloud-init/artifacts/cse_helpers.sh

```diff
@@ -384,40 +384,80 @@ should_skip_nvidia_drivers() {
     echo "$should_skip"
 }
 
-start_watch () {
-  capture_time=$(date +%s)
-  start_timestamp=$(date +%H:%M:%S)
+installJq () {
+  # jq is not available until downloaded in install-dependencies.sh with the installDeps function
+  # but it is needed earlier to call the capture_benchmarks function in pre-install-dependencies.sh
+  output=$(jq --version)
+  if [ -n "$output" ]; then
+    echo "$output"
+  else
+    if [[ $OS == $MARINER_OS_NAME ]]; then
+      sudo tdnf install -y jq
+      echo "jq was installed: $(jq --version)"
+    else
+      apt_get_install 5 1 60 jq
+      echo "jq was installed: $(jq --version)"
+    fi
+  fi
 }
 
-stop_watch () {
+capture_benchmarks () {
```

Comment:
I just added a shell script unit test and merged to master branch. Please git pull master branch and take a look at the section `shell scripts unit tests` in the readme. It will be great if you can add unit tests to your new functions. Examples are listed in the readme too.